host/azure-pipelines.yml

```diff
@@ -17,8 +17,55 @@ pr:
     include:
       - host/src/
 stages:
+  - stage: BuildAndPublishPrelaunchApps
+    displayName: "Publish pre-launch app for NET 8.0 and NET 7.0"
+    jobs:
+      - job: PublishPrelaunchApp
+        displayName: "Publish Prelaunch apps"
+        pool:
+          vmImage: "windows-latest"
```

Comment:
Curious why this is windows? Is it important? If we were to use ubuntu, you could avoid having this be a separate stage at all and do it as part of the existing functions net host stage.

Reply:
Could be any platform (Windows or Linux). I just picked Windows for no particular reason. We are publishing the apps as portable. FNH is published as both Windows and Linux. I want to keep this as a separate stage for readability purposes.